.github/workflows/ci.yml

```diff
@@ -0,0 +1,40 @@
+name: Gsutil CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [pypy2, 2.7, pypy3, 3.5, 3.6, 3.7]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        set -x
+        pip install .
+        git submodule update --init --recursive
```

Comment:
Pip should install all the dependencies. Do we still need git submodule update?

Reply:
Error without:
```
+ python gsutil version -l
Traceback (most recent call last):
  File "gsutil", line 21, in <module>
    gsutil.RunMain()
  File "/home/runner/work/gsutil/gsutil/gsutil.py", line 121, in RunMain
    import gslib.__main__
  File "/home/runner/work/gsutil/gsutil/gslib/__main__.py", line 94, in <module>
    from gslib.command_runner import CommandRunner
  File "/home/runner/work/gsutil/gsutil/gslib/command_runner.py", line 64, in <module>
    from gslib.tests.util import HAS_NON_DEFAULT_GS_HOST
  File "/home/runner/work/gsutil/gsutil/gslib/tests/util.py", line 38, in <module>
    import mock_storage_service  # From boto/tests/integration/s3
ModuleNotFoundError: No module named 'mock_storage_service'
```